netbox/ipam/models/vlans.py

```diff
@@ -45,23 +52,16 @@ class VLANGroup(OrganizationalModel):
         ct_field='scope_type',
         fk_field='scope_id'
     )
-    min_vid = models.PositiveSmallIntegerField(
-        verbose_name=_('minimum VLAN ID'),
-        default=VLAN_VID_MIN,
-        validators=(
-            MinValueValidator(VLAN_VID_MIN),
-            MaxValueValidator(VLAN_VID_MAX)
-        ),
-        help_text=_('Lowest permissible ID of a child VLAN')
+    vlan_id_ranges = ArrayField(
+        BigIntegerRangeField(),
+        verbose_name=_('min/max VLAN IDs'),
+        default=get_default_vlan_ids,
```

Comment:
Does this field need a default value?

Reply:
It was returning an error on a comparison or something, forget the exact error.